extensions/wikia/TemplateClassification/TemplateClassification.setup.php

```diff
@@ -101,3 +101,8 @@
 JSMessages::registerPackage( 'TemplateClassificationHints', [
 	'template-classification-bulk-classification-*'
 ] );
+
+JSMessages::registerPackage( 'TemplateClassificationGlobalShortcuts', [
+	'template-class-global-shortcuts-caption-classify-page',
+	'template-class-global-shortcuts-press-to-explore-shortcuts',
```

Comment:
I want to move it to TemplateClassification as it's used there, but I think it's better to wait for translation process to complete and then move.